### PR TITLE
delegation - unstake instructions

### PR DESCRIPTION
**Pull request by jimmy3dita**

### CODE REVIEW

File: docs/tokens/lockup.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+The most common configuration of lockup on NEAR, particularly for early holders, is to linearly release the tokens for transfer during the entire term of the lockup. For example, a 24 month linear lockup would make a small amount of tokens eligible for transfer with each block that passes until the full amount is free to transfer at the end of 24 months.
+
+Another factor in lockups is the "cliff", which means that no tokens are unlocked until that date (often 12 months after the lockup start).  On that date, a large chunk of tokens is unlocked at once to make it as if the cliff never existed at all.  Most early accounts are subject to a cliff. For example, a 4 year linear lockup with a 1 year cliff will have the following characteristics:
```
Comment (bowenwang1996) [inline docs/tokens/lockup.md:16]: lockup normally has no cliff. Only those with vesting have a cliff.

File: docs/tokens/lockup.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+1. Months 0-12: all tokens are locked
+2. Month 12+1 block: the first 25% of the tokens are immediately unlocked
+3. Months 13-48: the remaining 75% of tokens are unlocked smoothly over each block of the remaining 36 months.
+4. Months 48+: all tokens are unlocked
+
+The "start date" of lockups is the date that transfers are enabled on the network ("Phase II") for any contracts distributed prior to Phase II.  For any lockup contracts implemented after this date, it will be simply the date the lockup contract was implemented.
```
Comment (bowenwang1996) [inline docs/tokens/lockup.md:23]: this is not always the case. Some have an absolute start date that comes before the start of phase 2.

File: docs/tokens/lockup.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+There are three ways to go:
+ - Check your balance using https://near.github.io/account-lookup/.
+ - [Import your account into NEAR Wallet](token-custody#importing-accounts-from-other-wallets).
+ - Use CLI to check your balance: `near view <LOCKUP_ACCOUNT_ID> get_balance ''` (note it outputs the value in yoctoNEAR - divide by 10e24 to get NEAR amount).
+
+### How do I delegate locked up tokens?
+
+It's important to note, that a single lockup contract can only delegate to a single validator.
+
+Go to Dokia's staking UI: https://staking.dokia.cloud/staking/near/validators
```
Comment (bowenwang1996) [inline docs/tokens/lockup.md:113]: we should also link to instructions on how to do this through near-cli. Also I am not sure whether we want to endorse dokia's portal officially.

**Comment by jimmy3dita** [comment]
I'm merging the contents by now, as the lockup pages were initially created by @ilblackdragon and I only fixed the 404 links.

**Pull request merged**